doc/RecordingControl.xml

```diff
@@ -2089,13 +2094,13 @@ secfrac = "." 1*6DIGIT</programlisting>
         </mediaobject>
       </figure>
     </section>
-    <section xml:id="_refCloudRecordingEncryption">
-      <title>Encryption</title>
+    <section xml:id="_refCloudRecordingFrameEncryption">
+      <title>Frame encryption</title>
       <para>
-        A device signaling support for recording to an external target with encryption shall support writing encrypted files according to ISO/IEC 23001-7 (common encryption in ISO base media file format files).
+        A device signaling support for recording with frame encryption via EncryptionEntryLimit shall support creating encrypted files according to ISO/IEC 23001-7 (common encryption in ISO base media file format files).
```

Comment:
Suggest to rephrase to ```A device signaling support for encrypted recording via the EncryptionLimit capability ...```.

It may also be better to tie it to SupportedEncryptionModes and let the EncryptionLimit default to one.